src/auto_pickup.cpp

```diff
@@ -78,7 +78,9 @@ void auto_pickup::show( const std::string &custom_name, bool is_autopickup )
     tmpx += shortcut_print(w_header, 0, tmpx, c_white, c_ltgreen, _("<M>ove")) + 2;
     tmpx += shortcut_print(w_header, 0, tmpx, c_white, c_ltgreen, _("<E>nable")) + 2;
     tmpx += shortcut_print(w_header, 0, tmpx, c_white, c_ltgreen, _("<D>isable")) + 2;
-    shortcut_print(w_header, 0, tmpx, c_white, c_ltgreen, _("<T>est"));
+    if( g->u.name != "" ) {
```

Comment:
This is a minimal bugfix so builds don't crash; I didn't want to refactor further.